ivy_tests/test_ivy/test_frontends/test_torch/test_indexing_slicing_joining_mutating_ops.py

```diff
@@ -113,6 +113,47 @@ def test_torch_concat(
     )
 
 
+# nonzero
+@handle_cmd_line_args
+@given(
+    dtype_and_values=helpers.dtype_and_values(
+        available_dtypes=tuple(
+            set(ivy_np.valid_float_dtypes).intersection(
+                set(ivy_torch.valid_float_dtypes)
+            )
+        ),
+        shape=st.shared(helpers.get_shape(min_num_dims=1), key="shape"),
```

Comment:
Sorry, I'm not familiar with the codebase so I'm not quite sure what you mean by that. I added that bit of code (line 125) because I wanted to ensure that the test inputs have a dimension of 1 or more, because an input with 0 dimensions would be invalid (hence the `min_num_dims=1`). I'm not sure what you mean by the shape not being used.

Reply:
Yea my bad I misread this as an extra parameter for the testing not a parameter for `dtypes_and_values`. In this case there is a parameter for `dtypes_and_values`, `min_num_dims` that you can use rather than setting the shape parameter

Reply:
Thanks for letting me know, I've now updated the test to use the `min_num_dims` parameter rather than setting the shape parameter.